man/systemd.network.xml

```diff
@@ -1121,6 +1121,62 @@ Table=1234</programlisting></para>
           Defaults to <literal>no</literal>.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetLabel=</varname><replaceable>label</replaceable></term>
+        <listitem>
+
+          <para>This setting provides a method for integrating static and dynamic network configuration into
+          Linux <ulink url="https://docs.kernel.org/netlabel/index.html">NetLabel</ulink> subsystem rules,
+          used by <ulink url="https://en.wikipedia.org/wiki/Linux_Security_Modules">Linux Security Modules
+          (LSMs)</ulink> for network access control. The label, with suitable LSM rules, can be used to
+          control connectivity of (for example) a service with peers in the local network. At least with
+          SELinux, only the ingress can be controlled but not egress. The benefit of using this setting is
```

Comment:
does any LSM other than SELinux actually care about NetLabel?

Sounds to me, if the answer to that question is "no", we shouldn't mention LSM here, and just say SELinux.

Reply:
Smack [seems to care](https://wiki.tizen.org/Security:SmackNetworking) but the interface may be different. If so, it would need some work to add Smack support (by someone knowledgeable). I suppose the concept of NetLabel configuration by networkd still makes sense. Perhaps the documentation should warn that only SELinux is implemented now.

Reply:
Added a note that SELinux is only implemented